### ISSUE TITLE
Created medium is not bootable.

### ISSUE THREAD

**Issue by KarelHaruda** [inindev/odroid-m1#7]
I followed these 3 steps.
1. download the image

wget https://github.com/inindev/odroid-m1/releases/download/v12.0.2/odroid-m1_bookworm-1202.img.xz

2. determine the target micro sd card location

before connecting the device
ls -l /dev/sd*
ls: cannot access '/dev/sd*': No such file or directory
after connecting the device
ls -l /dev/sd*
brw-rw---- 1 root disk 8, 0 Apr 10 15:56 /dev/sda
note: for Mac the device is /dev/rdiskX

3. in the above case replace 'a' with 'X' in the command below (for /dev/sda)

sudo sh -c 'xzcat odroid-m1_bookworm-1202.img.xz > /dev/sdX && sync'

After all these steps, Odroid cannot boot from this SD card
Is there anything else that needs to be done?

Thank you
Karel

**Comment by inindev** [comment]
It is probably trying to boot from the spi nor memory.

Insert the mmc media and the hold the spi flash bypass button while powering on the device. The button can be seen near the 8 pin flash chip at the front of the board.
https://wiki.odroid.com/_media/odroid-m1/odroidm1nvme.jpg

Note: The mmc media has a one-time reboot during first setup as it expands to the size of the mmc media. Without a serial terminal it will be difficult to know when this reboot happens. Waiting two minutes then powering down and booting again with the spi bypass button depressed again should be sufficient to reach the second boot.

Here is the vendor doc on booting: https://wiki.odroid.com/odroid-m1/software/boot_sequence

<br/>
Also, if flashing the mmc is successful, you should be able to mount its first partition on another linux machine.

I have been breaking this one-liner up into multiple lines in the doc I wrote for some of the other boards:
```
sudo su
xzcat odroid-m1_bookworm-1202.img.xz > /dev/sdX
sync
```

**Comment by KarelHaruda** [comment]
When I tried to hold that button while turning on the power, Odroid won't even start. The power-on LED doesn't even light up.
But that doesn't matter, because I found that if I use the netboot_default command in Petitboot, it will offer to install Debian 12 directly from the Internet, so I don't have to go around it anymore.
Anyway, after I wrote the IMG to the SD card, the first partition can actually be mounted and Debina 12 is indeed there.
However, I install it using netboot_default

Thank you for your help and we can close this
